lix/packages/sdk/src/branch/errors.ts

```diff
@@ -0,0 +1,18 @@
+import dedent from "dedent";
+
+export class BranchError extends Error {
+	constructor(message: string) {
+		super(message);
+		this.name = "BranchError";
+	}
+}
+
+export class BranchExistsError extends BranchError {
+	constructor(args: { name: string }) {
+		super(dedent`
+The branch with id "${args.name}" already exists.
```

Comment:
```suggestion
	constructor(args: { id: string }) {
		super(dedent`
The branch with id "${args.id}" already exists.
```

Reply:
The `name` has been used as `id` here. That's a bug. If this is a user facing error, adding both id and name to the error makes sense. 

- apps can process the branch id 
- users see an error message for the branch name

```
constructor(args: { id: string, name: string }){
  this.id = args.id
  this.name = args.name
  this.message = `The branch "${args.name}" already exists.`
}
```